Makefile

```diff
@@ -0,0 +1,28 @@
+init:
+	composer install --no-interaction --no-scripts
+	bin/console sylius:install --no-interaction
+	bin/console sylius:fixtures:load default --no-interaction
+	yarn install --pure-lockfile
+	yarn build
+
+ci:
+	composer install --no-interaction --no-scripts
+	bin/console sylius:install --no-interaction
+	bin/console sylius:fixtures:load default --no-interaction
+	bin/console cache:warmup
+	yarn install --pure-lockfile
+	yarn build
+	vendor/bin/phpunit
+	vendor/bin/phpspec run --ansi --no-interaction -f dot
+	vendor/bin/behat --colors --strict --no-interaction -vvv -f progress --tags="~@javascript&&@cli&&~@todo"  # CLI Behat
```

Comment:
Thought for later - won't it be better to reference this file instead of duplication of logic?